compiler/il/OMRDataTypes.cpp

```diff
@@ -158,6 +158,15 @@ OMR::DataType::getSize(TR::DataType dt)
             TR_ASSERT_FATAL(false, "Incorrect Vector Length\n");
          }
       }
+   else if (dt.isMask())
+      {
+#if defined(TR_TARGET_POWER) || defined(TR_TARGET_S390)
```

Comment:
Can you please add `TR_TARGET_ARM64` here?

Reply:
Done